src/components/forms/AddRecipientForm.js

```diff
@@ -106,34 +134,45 @@ function AddRecipientForm({ visible, onCreate, onCancel, households }) {
             <Select
               // mode="multiple" - limiting race to 1 temporarily to test Create Recipient
               placeholder="Please select the race of the recipient"
+              rules={[{ required: true }]}
             >
-              <Option value="indian_native_alaskan">
-                American Indian or Alaska Native
-              </Option>
-              <Option value="asian">Asian</Option>
-              <Option value="black">Black/African American</Option>
-              <Option value="hawaiian_pacific_islander">
-                Native Hawaiian or Pacific Islander
-              </Option>
-              <Option value="white">White/Caucasian</Option>
+              <Option value={1}>White</Option>
+              <Option value={2}>Black/African-American</Option>
+              <Option value={3}>Asian</Option>
+              <Option value={4}>Native Americian/First Peoples</Option>
+              <Option value={5}>Hawaiian/Pacific Islander</Option>
+              <Option value={6}>Other</Option>
+              <Option value={7}>Prefer not to say</Option>
             </Select>
           </Form.Item>
-          <Form.Item label="Select Ethnicity" name="ethnicity">
+          <Form.Item
+            label="Select Ethnicity"
+            name="ethnicity_id"
+            rules={[{ required: true }]}
+          >
             <Radio.Group>
-              <Radio value="hispanic">Hispanic or Latino</Radio>
-              <Radio value="not_hispanic">Not Hispanic or Latino</Radio>
+              <Radio value={1}>Hispanic/Latino</Radio>
+              <Radio value={2}>Non-Hispanic/Non-Latino</Radio>
+              <Radio value={3}>Prefer not to say</Radio>
             </Radio.Group>
           </Form.Item>
-          <Form.Item label="Select Veteran Status" name="veteran_status">
+          <Form.Item
+            label="Select Veteran Status"
+            name="recipient_veteran_status"
+            rules={[{ required: true }]}
+          >
             <Radio.Group>
               <Radio value="true">Veteran</Radio>
               <Radio value="false">Not a Veteran</Radio>
             </Radio.Group>
           </Form.Item>
```

Comment:
Will need to add forms for `has_disability`, `has_valid_ssi`, and `has_valid_medicare_card` boolean columns (identical to veteran status) in a future PR.